### DISCUSSION TITLE
Filter hosts that open ports that exceed a certain threshold

### DISCUSSION THREAD

**Discussion by 0cat-r** [projectdiscovery/naabu#367] [Ideas]
Is it possible to add a parameter that a host opens a certain threshold number of ports to filter out to scan the next target.
For example, if an ip has opened 100 ports, then it will not continue to scan this ip.

**Comment by ehsandeep** [comment]
It will be supported in future releases as part of https://github.com/projectdiscovery/naabu/issues/294 @0cat-r

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
